src/Equinox.Cosmos/Cosmos.fs

```diff
@@ -939,15 +912,15 @@ type private Folder<'event, 'state>
         ?readCache) =
     let inspectUnfolds = match mapUnfolds with Choice1Of3 () -> false | _ -> true
     interface ICategory<'event, 'state, Container*string> with
-        member __.Load containerStream (log : ILogger): Async<StreamToken * 'state> =
-            let batched = category.Load inspectUnfolds containerStream fold initial isOrigin log
+        member __.Load containerStream (log : ILogger): Async<StreamToken * 'state>  = async {
+            let! batched = category.Load inspectUnfolds containerStream fold initial isOrigin log
```

Comment:
this is a behaviour change - should be return! on L922 - same in other callsite

Reply:
Totally missed that 🤦‍♂. Looks like this also caused an integration test failure.

Reply:
Yeah, I've been 🙈about automating the running of those for too long; Unfortunately it's not in danger of getting to the top of my TODO list for a while :(